main.go

```diff
@@ -69,10 +73,24 @@ func main() {
 		log.Printf("%s: reading env file %q: %v", logTag, envFile, err)
 	}
 
+	if Billing {
```

Comment:
@bietkul Instead of taking Billing as a flag, we can remove the runtime flag. And pass a build time variable value via -ldflags like mentioned over here: https://blog.alexellis.io/inject-build-time-vars-golang/.

> 	$(GC) -ldflags "-X main.Billing=${BILLING}" -o $(BUILD_DIR)/arc main.go

Reply:
The main.go file change will be merged in `appbaseio/arc:dev` project.